app/code/Magento/Checkout/Block/Cart/Crosssell.php

```diff
@@ -133,9 +132,7 @@ public function getItems()
                 if ($lastAddedProduct) {
                     $collection = $this->_getCollection()
                         ->addProductFilter($lastAddedProduct->getData($this->getProductLinkField()));
-                    if (!empty($ninProductIds)) {
-                        $collection->addExcludeProductFilter($ninProductIds);
-                    }
+                    $collection->addExcludeProductFilter($ninProductIds);
```

Comment:
In the Adobe "Static Tests" check (https://github.com/magento/magento2/runs/11822411725), PHPStan reported:
```
 ------ -------------------------------------------------------------------- 
  Line   app/code/Magento/Checkout/Block/Cart/Crosssell.php                  
 ------ -------------------------------------------------------------------- 
  135    Variable $ninProductIds in empty() always exists and is not falsy.  
 ------ -------------------------------------------------------------------- 
```
see https://public-results-storage-prod.magento-testing-service.engineering/reports/magento/magento2/pull/36991/ad3212c981ae968b07405b9fecfbf14c/Statics/console-error-logs.html

To avoid the test failure, I adjusted the code to be more sensible.